### PR TITLE
Fix gravity task

### PR DESCRIPTION
**Pull request by davidegorbani**
With this PR I fixed the bug in the gravity task; basically, the gravity task implemented in `blf` requires the direction vector to be in body-frame while the one that was set was in world-frame. Moreover, I fixed the `t-pose` calibration of the feet; now, when the `t-pose` is performed, the setpoint of the floor contact task is recomputed starting from the `t-pose` position.

### CODE REVIEW

**Comment by davidegorbani** [comment]
The macos CI is failing, I compared the `micromamba list` of the failing CI with one of a successful CI and I noticed that `qt6-main` is missing in the failing CI; should I add it as a dependency in [here](https://github.com/ami-iit/biomechanical-analysis-framework/blob/main/ci_env.yml) @traversaro?

File: src/IK/src/InverseKinematics.cpp [hunk 1]
```diff
@@ -201,7 +201,7 @@ bool HumanIK::updateGravityTask(const int node, const manif::SO3d& I_R_IMU)
 
     // set the set point of the gravity task choosing the z direction of the W_R_link rotation
     // matrix
```
Comment (traversaro) [inline src/IK/src/InverseKinematics.cpp:203]: Should we update the comment as well?

**Comment by traversaro** [comment]
> The macos CI is failing, I compared the `micromamba list` of the failing CI with one of a successful CI and I noticed that `qt6-main` is missing in the failing CI; should I add it as a dependency in [here](https://github.com/ami-iit/biomechanical-analysis-framework/blob/main/ci_env.yml) @traversaro?

I actually think you just need to set the env variable `QT_HOST_PATH` to `$CONDA_PREFIX` just on macos, this is due to https://github.com/conda-forge/qt-main-feedstock/issues .

**Review by Gianlucamilani — APPROVED** [review]

**Comment by davidegorbani** [comment]
Closing as the CI is fixed now; thank you @traversaro @Gianlucamilani!

**Pull request merged**